api/src/main/java/org/pmiops/workbench/utils/mappers/LeonardoMapper.java

```diff
@@ -197,15 +197,17 @@ default void listRuntimeAfterMapper(
         leonardoListRuntimeResponse.getDiskConfig());
   }
 
-  @Mapping(target = "createdDate", source = "app.auditInfo.createdDate")
-  @Mapping(target = "dateAccessed", source = "app.auditInfo.dateAccessed")
+  @Mapping(target = "createdDate", source = "auditInfo.createdDate")
```

Comment:
is this 3 lines duplicate with line 208-210 below?

Reply:
This is for the get request, lines 208-210 for the list request. app.auditInfo.createdDate is not right because it doesn't map correctly, that's why I fixed it